### ISSUE TITLE
[bug]: `ringSector` plugin issue with `center` parameter

### ISSUE THREAD

**Issue by woutervdub** [freesewing/freesewing#5336]
### What seems to be the problem? 🤔

If the `center` point does not have an `x` value of 0, the result is not a circle segment:
![image](https://github.com/freesewing/freesewing/assets/24414629/3e3f80cc-a1c2-4a87-b720-96a592ee8b09)

### Design / Plugin / Package 🧐

plugin-ringSector

### Are you a FreeSewing patron? 😃

Yes, I am a tier-2 patron ❤️

**Comment by woutervdub** [comment]
Also found: The `path` is in reverse. 

You have to do this to add `sa`:
```javascript
    if (sa) paths.sa = paths.__macro_ringsector__path.reverse().offset(sa).attr('class', 'fabric sa')
```

Documentation mentions `id` twice.

**Comment by joostdecock** [comment]
For the SA issue, you don't have to `reverse` the path, you can just `.offset(-1 * sa)`

**Comment by joostdecock** [comment]
Fixed by @woutervdub in #5248
